src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -555,7 +518,10 @@ class BinaryCompactObject : public DomainCreator<3> {
   bool is_excised_b_ = false;
   bool use_single_block_a_ = false;
   bool use_single_block_b_ = false;
-  std::optional<bco::TimeDependentMapOptions> time_dependent_options_{};
+  // Mutable because we have to set some parameters in the time_dep_options in
+  // order to construct the maps
+  // NOLINTNEXTLINE
+  mutable std::optional<bco::TimeDependentMapOptions> time_dependent_options_{};
```

Comment:
Any suggestions for an alternative to this would be appreciated.

Reply:
Construct the maps in the TimeDepOptions constructor so you can copy them later, or construct the maps on-demand since copying them has little benefit over constructing.